### PR TITLE
Boost: Update changelogs and readmes for 1.3.1-beta

### PR DESCRIPTION
**Pull request by thingalon**
Prepare for a 1.3.1 beta release by updating changelogs and readmes.

Also updates dependencies.

#### Changes proposed in this Pull Request:
* Update changelogs and readmes.

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
Yes; it adds the following data to our API requests when fetching Speed Scores, which are stored along with PageSpeed results:

- Jetpack Boost version.
- Which Jetpack Boost modules are enabled.

The purpose of this data is to give us a statistical overview of the effect each module (and version) has on speed scores.

#### Testing instructions:
* Check that the changelogs look ok.
* Check that the dependency changes are ok.

### CODE REVIEW

File: projects/js-packages/webpack-config/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.2.1 - 2021-11-19
```
Comment (anomiex) [inline projects/js-packages/webpack-config/CHANGELOG.md:8]: We should have something in the section here. Did changelogger somehow leave this empty, or did you manually delete the "Updated package dependencies" entry?

**Review by anomiex — COMMENTED** [review]
Also, a small request: It'd be more clear in notifications and lists what this PR applies to if "Boost" were in the title somewhere. Like "Boost: Update changelogs and readmes for 1.3.1-beta" or "Update changelogs and readmes for Boost 1.3.1-beta".

File: projects/js-packages/webpack-config/CHANGELOG.md (hunk 1 above)
Comment (thingalon) [reply projects/js-packages/webpack-config/CHANGELOG.md:8]: The changelogger left it empty; the only changelog I manually edited was Boost's. 

I've inserted a changelog entry there manually for updating project dependencies in 3e4b92a1b. Thanks for picking up on this.

**Comment by thingalon** [comment]
This has gotten out of sync with master package files / versions of things have moved. Easier to delete the branch and recreate at this point.

**Comment by thingalon** [comment]
Replaced with #21839 which has a fresh branch with freshly generated changelogs.

**Pull request closed without merging**